### PR TITLE
Points to more information on labeled arguments

### PR DESCRIPTION
**Pull request by prometheansacrifice**
Helps a beginner make sense of the example

### CODE REVIEW

**Comment by prometheansacrifice** [comment]
Personally experienced it. I had to ping the discord channel asking what it was.

File: src/pages/guide/language/types.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ let myInt = 5;
 let myInt = (5 : int);
 let myInt = (5 : int) + (4 : int);
 let add (x: int) (y: int) :int => x + y;
-let drawCircle radius::(r: int) :unit => ...;
+let drawCircle radius::(r: int) :unit => ...; /* radius::(r: int) is a labeled argument. More on this [here](https://reasonml.github.io/guide/language/functions) */
```
Comment (prometheansacrifice) [inline src/pages/guide/language/types.md]: Ideally I wish we could have fragment identifiers for subheadings. Labeled arguments could be mentioned directly.

Comment (jaredly) [inline src/pages/guide/language/types.md]: +1 we'll be adding that in a bit

**Comment by jaredly** [comment]
thanks!

**Pull request merged**